server/services/companies/companies-webhook.service.ts

```diff
@@ -13,11 +14,33 @@ class CompaniesWebhookService {
     // TODO: Authenticate against Typeform.
     const payload = data.form_response;
 
+    // Override team selection if company selects random
+    const videoConferenceAnswer = getAnswerValueFromRef(
+      payload,
+      refsMap.video_conference
+    );
+    const isRemoteApplication =
+      videoConferenceAnswer === 'Via video conference';
+
+    let team;
+    if (isRemoteApplication) {
+      const choices = [
+        Team.Boston,
+        Team.Philadelphia,
+        Team.NewYork,
+        Team.SanFrancisco,
+      ];
+      const choice = Math.floor(Math.random() * 3); // returns a random integer from 0 to 3
```

Comment:
Should be Math.floor(Math.random() * 4); for vals in [0,3]

Reply:
Good catch thanks.